App.xaml.cs

```diff
@@ -1,12 +1,17 @@
 ﻿
+using System.Security.Cryptography.X509Certificates;
 using Combat_Critters_2._0.Pages;
+using CombatCrittersSharp;
 
 namespace Combat_Critters_2._0
 {
 	public partial class App : Application
 	{
+		//Store the current user's client instance globally
+		public static IClient? CurrentClient {get; set;}

```

Comment:
this should not be publicly set, Client should be initialized in App and then passed as a constructor arg to the BackendService which probably shouldnt be static.

alternatively Backend service can be a singleton and initializes the client in its constructor.

additionally keep in mind that at some point (this iteration) we will need to inject the API URI rather than hardcoding it.